crates/aranya-client/tests/tests.rs

```diff
@@ -290,8 +290,8 @@ impl Drop for UserCtx {
 /// `RUST_LOG="aranya_client=debug,aranya_daemon=info" cargo test integration_test -- --show-output --nocapture`
 #[test(tokio::test(flavor = "multi_thread"))]
 async fn integration_test() -> Result<()> {
-    let sync_interval = Duration::from_millis(100);
-    let sleep_interval = sync_interval * 6;
+    let sync_config: SyncPeerConfig = Default::default();
```

Comment:
Can use shorter variables names:
```suggestion
    let cfg = SyncPeerCfg::default();
```